.travis.yml

```diff
@@ -117,6 +117,22 @@ jobs:
           tags: false
           repo: triplea-game/triplea
           branch: master
+    - stage: deploy prerelease
+      if: (branch = master) and (repo = 'triplea-game/triplea') and (type != 'pull_request')
+      install:
+        - sudo apt update
+        - sudo apt -y install sshpass
```

Comment:
I don't think travis is allowed to run sudo commands, I think you need to declare the packages as usual (see other stages)

Reply:
AFAIK travis can run sudo commands. By default even it'll run all commands as sudo. We have it disabled though: https://github.com/triplea-game/triplea/pull/5556/files/3d25ae1aa1091c4e4c02179f72e33428bcae8f2c#diff-354f30a63fb0907d4ad57269548329e3R2

Sudo is allowed for all Travis OS except windows: https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system

I checked if we could get away without the sudo on either apt update or install, we need it on both:
-  https://travis-ci.org/DanVanAtta/triplea/jobs/615871270
- https://travis-ci.org/DanVanAtta/triplea/jobs/615865889

I don't know why sudo is not needed on the other apt install commands, perhaps they are unnecessary and those packages are installed? I'm not sure.